nex/internals/rootfs.md

```diff
@@ -2,6 +2,27 @@
 The root file system used by Nex in its spawned Firecracker virtual machines is an ext4 (64-bit file system) [block device](https://linux-kernel-labs.github.io/refs/heads/master/labs/block_device_drivers.html). In oversimplified terms, it's basically a single file that represents an entire file system.
 
 ## Building a Root File System
+#### Using `nex` CLI
+We have added the ability to build a root filesystem to the Nex CLI.  To build a root filesystem, run the following command:
```

Comment:
Suggest the following since the docs don't have versions - "As of April 2024, Nex CLI has the ability to build a root filesystem."